src/web/app/pages-instructor/instructor-course-details-page/__snapshots__/instructor-course-details-page.component.spec.ts.snap

```diff
@@ -0,0 +1,422 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorCourseDetailsPageComponent should snap with a course with one co-owner and no students 1`] = `
+<tm-instructor-course-details-page
+  clipboardService={[Function ClipboardService]}
+  courseDetails={[Function Object]}
+  courseService={[Function CourseService]}
+  courseStudentListAsCsv=""
```

Comment:
Might want to include a case with this populated so we know it is rendered correctly